packages/editor-ui/src/composables/useToast.ts

```diff
@@ -20,7 +20,7 @@ export interface NotificationErrorWithNodeAndDescription extends ApplicationErro
 const messageDefaults: Partial<Omit<NotificationOptions, 'message'>> = {
 	dangerouslyUseHTMLString: false,
 	position: 'bottom-right',
-	zIndex: 1900, // above NDV and below the modals
+	zIndex: 2100, // above NDV and modal overlays
```

Comment:
Can we maintain all various z-index values as `const enum` in a single place? That way it'll be easier to keep track of all the visual layers in app.

Reply:
I thought about that but how would you handle ones set as css?

Reply:
we could keep a list of css variables or always try to use js

Reply:
**any** way that can help us keep track of all the major layers in a single place works for me. I'm indifferent about the actual implementation, as long as we can debug issues like this by looking in a single place in the codebase.

Reply:
makes sense. I think I can do it by generating css vars from an object.